examples/seismic/acoustic/wavesolver.py

```diff
@@ -167,8 +169,25 @@ def gradient(self, rec, u, v=None, grad=None, m=None, **kwargs):
         if m is None:
             m = m or self.model.m
 
-        summary = self.op_grad().apply(rec=rec, grad=grad, v=v, u=u, m=m,
-                                       dt=kwargs.pop('dt', self.dt), **kwargs)
+        if checkpointing:
+            set_log_level('ERROR')
+            u = TimeFunction(name='u', grid=self.model.grid,
+                             time_order=2, space_order=self.space_order)
+            cp = DevitoCheckpoint([u])
+            n_checkpoints = None
```

Comment:
what is this variable?

Reply:
Number of checkpoints to save, None lets PyRevolve decide